### PR TITLE
feat(conda_perl): Using conda version 4.9.1 this will install all MIP…

### PR DESCRIPTION
**Pull request by henrikstranneheim**
… perl dependencies without errors using only cpanm

### This PR adds:

- See above. Placing this file here for future reference. Showing that using an updated version of conda the installation actually runs quite smooth.

### How to test:

- Automatic and continuous test suite

### Expected outcome:
- [ ] Installation, unit and integration test suite pass

### Review:
- [ ] Code review
- [ ] Tests pass

This [version](https://semver.org/) is a:
- [ ] **MAJOR** - when you make incompatible API changes. If applicable record manual test results in PR header
- [ ] **MINOR** - when you add functionality in a backwards compatible manner. If applicable record manual test results in PR header
- [x] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

File: templates/mip_install_perl_conda-v4.9.1.sh [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+if [ "$DEV" = true ]; then
+
+    ## Install dev modules from cpan
+    cpanm Perl::Tidy@20200110 Perl::Critic@1.138 Data::Printer@0.40
+
+    ## Install yamllint
+    conda install --name "$ENV_NAME" --yes -c conda-forge yamllint=1.20.0
+fi
+
+#conda deactivate
```
Comment (jemten) [inline templates/mip_install_perl_conda-v4.9.1.sh:87]: remove?

**Review by jemten — APPROVED** [review]
Do we need to update conda on hasta for this to work?

**Comment by henrikstranneheim** [comment]
> Do we need to update conda on hasta for this to work?

Yes, this is more for bookkeeping that this is all that should be required with well updated and rather clean conda.

File: templates/mip_install_perl_conda-v4.9.1.sh (hunk 1 above)
Comment (henrikstranneheim) [inline templates/mip_install_perl_conda-v4.9.1.sh:87]:
```suggestion
conda deactivate
```

Comment (henrikstranneheim) [reply templates/mip_install_perl_conda-v4.9.1.sh]: Removed comment symbol

**Pull request merged**